datajoint/settings.py

```diff
@@ -93,6 +93,18 @@ def __iter__(self):
     def __len__(self):
         return len(self.instance._conf)
 
+    def save_local(self):
+        """
+        saves the settings in the local config file  
+        """
+        self.save(LOCALCONFIG)
+
+    def save_global(self):
+        """
+        saves the settings in the global config file  
+        """
+        self.save(os.path.expanduser('~/' + GLOBALCONFIG))
```

Comment:
To be safe, I recommend that we use `path.join` from `os.path` package when combining paths.